src/model/model.js

```diff
@@ -296,15 +296,44 @@ export default class Model {
 	 *		// which has a loosened schema.
 	 *		const modelFragment = editor.data.toModel( viewFragment );
 	 *
-	 *		editor.model.insertContent( modelFragment, editor.model.document.selection );
+	 *		editor.model.insertContent( modelFragment );
+	 *
+	 * By default the method will use document selection but it can be also used with position, range or selection instances.
+	 *
+	 *		// Insert text at current document selection position.
+	 *		editor.model.change( writer => {
+	 *			editor.model.insertContent( writer.createText( 'x' ) );
+	 *			// equal to editor.model.insertContent( model, writer.createText( 'x' ), editor.model.document.selection );
```

Comment:
I think it's a bit unnecessary and makes it more visually complex. Plus, there's a mistake (the `model` param).